chrome/content/zotero/integration/citationDialog/searchHandler.mjs

```diff
@@ -195,7 +195,13 @@ export class CitationDialogSearchHandler {
 
 	cleanSearchQuery(str) {
 		str = str.replace(/ (?:&|and) /g, " ", "g").replace(/^,/, '');
-		str = this._cleanYear(str);
+		let issn = Zotero.Utilities.cleanISSN(str);
```

Comment:
An ISSN identifies a publication, not an item, so that shouldn't be here.

Reply:
Ah, sorry about that. I removed it and repushed the commit to fix the message.